### PR TITLE
Goodbye User Voice Blog

### PR DESCRIPTION
**Pull request by waderyan**
Any final tweaks? 

@chrisdias @egamma @gregvanl 

Also, timeline for release?

### CODE REVIEW

File: blogs/2016/08/19/goodbyeuservoice.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+Order: 14
+TOCTitle: Goodbye User Voice
+PageTitle: Goodbye User Voice
+MetaDescription: VS Code is closing User Voice in favor of Github reactions.
```
Comment (gregvanl) [inline blogs/2016/08/19/goodbyeuservoice.md]: Use "Visual Studio Code" in the metadescription

File: blogs/2016/08/19/goodbyeuservoice.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+Order: 14
+TOCTitle: Goodbye User Voice
+PageTitle: Goodbye User Voice
+MetaDescription: VS Code is closing User Voice in favor of Github reactions.  
+Date: 2016-08-18
+ShortDescription: VS Code is closing User Voice in favor of Github reactions. 
+Author: Wade Anderson
+---
+
+# Goodbye User Voice, hello Github Reactions!
```
Comment (gregvanl) [inline blogs/2016/08/19/goodbyeuservoice.md]: GitHub

File: blogs/2016/08/19/goodbyeuservoice.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+Author: Wade Anderson
+---
+
+# Goodbye User Voice, hello Github Reactions! 
+
+August 15, 2016 by Wade Anderson, [@waderyan_](https://twitter.com/waderyan_)
+  
+## Feedback Channels
+
+VS Code's [User Voice site](https://visualstudio.uservoice.com/forums/293070-visual-studio-code) has always been the place for the community to suggest and vote on feature requests for VS Code. The ability to see the requests with the top votes or the most activity has allowed us to prioritize the Code backlog and deliver many of the top requests over the past 18 months. For example, we’ve added [Extensions](https://visualstudio.uservoice.com/forums/293070-visual-studio-code/suggestions/9181439-plugins), [Tabs](https://visualstudio.uservoice.com/forums/293070-visual-studio-code/suggestions/7752519-implement-tabs) and an [integrated terminal](https://visualstudio.uservoice.com/forums/293070-visual-studio-code/suggestions/7752357-integrated-terminal), all because of your feedback and participation in User Voice.
```
Comment (gregvanl) [inline blogs/2016/08/19/goodbyeuservoice.md]: Use "Visual Studio Code" for the first reference in a doc, then "VS Code" after that.

File: blogs/2016/08/19/goodbyeuservoice.md (hunk 1 above)
Comment (gregvanl) [inline blogs/2016/08/19/goodbyeuservoice.md]: Use "Visual Studio Code" in the MetaDescription.
Should be "GitHub" everwhere in the doc.

File: blogs/2016/08/19/goodbyeuservoice.md (hunk 3 above)
Comment (gregvanl) [inline blogs/2016/08/19/goodbyeuservoice.md]: Use "Visual Studio Code" for first reference in a topic then "VS Code" not "Code" after that.  We'll sort this out with Marketing but for now let's stick with the current convention.  We can bulk edit later if we really have to.

File: blogs/2016/08/19/goodbyeuservoice.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+## Github Reactions
+
+![github reactions animation](2016_08_19_github_reaction.gif)
+
+Recently, Github introduced the ability to [add reactions to issues](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments). This feature replaces the key value proposition of User Voice, which has been the ability to vote on a request. It is now possible for the community to not only comment on an issue in Github, but also provide a thumbs up/thumbs down vote. This means we can use the same tools and workflows to sort and manage feature requests as we do to manage bugs.
+  
+## "Retiring" User Voice
+
+As a result, we are “retiring” the [VS Code User Voice site](https://visualstudio.uservoice.com/forums/293070-visual-studio-code) and asking the community to submit and vote on issues in Github.  To ease the transition, we’ve decided to move the current top 50 user voice issues (see below) over the next few weeks. If a feature request is more closely aligned with an extension rather than the core VS Code product, we’ll  move the request to the appropriate repository.
```
Comment (gregvanl) [inline blogs/2016/08/19/goodbyeuservoice.md]: Remove "(see below)" if you decide not to show top 50 per another PR.

File: blogs/2016/08/19/goodbyeuservoice.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+  
+## "Retiring" User Voice
+
+As a result, we are “retiring” the [VS Code User Voice site](https://visualstudio.uservoice.com/forums/293070-visual-studio-code) and asking the community to submit and vote on issues in Github.  To ease the transition, we’ve decided to move the current top 50 user voice issues (see below) over the next few weeks. If a feature request is more closely aligned with an extension rather than the core VS Code product, we’ll  move the request to the appropriate repository. 
+ 
+If your request is not on this list or if you have a new request, please open a new Github issue following the guidelines [here](https://github.com/Microsoft/vscode/blob/master/CONTRIBUTING.md). After you create an issue, we will run it through our [issue tracking](https://github.com/Microsoft/vscode/wiki/Issue-Tracking) workflow.  
+The inbox tracker for the week assigns your issue to the team member who owns the relevant area of code. The owner will assign the appropriate label and ask clarifying questions if necessary. During planning we review the open issues and assign the ones we plan to work on for the current milestone.
```
Comment (gregvanl) [inline blogs/2016/08/19/goodbyeuservoice.md]: "the relevant area of product source code."

File: blogs/2016/08/19/goodbyeuservoice.md (hunk 1 above)
Comment (waderyan) [inline blogs/2016/08/19/goodbyeuservoice.md]: Done and done.

File: blogs/2016/08/19/goodbyeuservoice.md (hunk 2 above)
Comment (waderyan) [inline blogs/2016/08/19/goodbyeuservoice.md]: Done.

File: blogs/2016/08/19/goodbyeuservoice.md (hunk 3 above)
Comment (waderyan) [inline blogs/2016/08/19/goodbyeuservoice.md]: Sounds good. Done.

**Pull request merged**